### ISSUE TITLE
phases are string instead of enum when using GridAPPS-D connector

### ISSUE THREAD

**Issue by AAndersn** [PNNL-CIM-Tools/CIM-Graph#65]
phases are returned as enum SinglePhaseKind when using Blazegraph or RDFLib connection

phases are returned as string when using GridAPPS-D connection

Test script:

```
utils.get_all_line_data(network)
for phs in network.graph[cim.ACLineSegmentPhase].values():
    print(type(phs.phase))
```

**Comment by AAndersn** [comment]
@afisher1 -- this should be fixed in new PR

**Comment by AAndersn** [comment]
Fixed in #67
